src/pds/registrysweepers/repairkit/__init__.py

```diff
@@ -69,6 +73,10 @@ def generate_updates(
                     for func in funcs:
                         repairs.update(func(src, fieldname))
 
+                # TODO: remove me once applied -- edunn 20231206
```

Comment:
Should we do that before the nominal repair update ? And do we need to apply that in for for loop on REPAIR_TOOLS ?

Reply:
@tloubrieu-jpl good catch, tyvm!

In this case it makes no difference (to the behaviour of the temporary code) but I've pushed a fix.